docs/en/database_engines/mysql.md

```diff
@@ -0,0 +1,119 @@
+#MySQL
+
+Allows to exchange data with the remote MySQL server.
+
+You can connect to some database on the MySQL server and perform `INSERT` and `SELECT` queries with tables.
+
+MySQL database engine cannot perform the following operations with tables:
+
+- Attach/Detach
+- Remove
+- Rename
+- Create
+- Alter
+
+
+## Creating a Database
+
+``` sql
+CREATE DATABASE [IF NOT EXISTS] db_name [ON CLUSTER cluster]
+ENGINE = MySQL('host:port', 'database', 'user', 'password')
+```
+
+**Engine Parameters**
+
+- `host:port` — MySQL server address.
+- `database` — Remote database name.
+- `user` — MySQL user.
+- `password` — User password.
+
+
+## Data Types Support
+
+MySQL | ClickHouse
+------|------------
+UNSIGNED TINYINT | [UInt8](../data_types/int_uint.md)
+TINYINT | [Int8](../data_types/int_uint.md)
+UNSIGNED SMALLINT | [UInt16](../data_types/int_uint.md)
+SMALLINT | [Int16](../data_types/int_uint.md)
+UNSIGNED INT, UNSIGNED MEDIUMINT | [UInt32](../data_types/int_uint.md)
+INT, MEDIUMINT | [Int32](../data_types/int_uint.md)
+UNSIGNED BIGINT | [UInt64](../data_types/int_uint.md)
+BIGINT | [Int64](../data_types/int_uint.md)
+FLOAT | [Float32](../data_types/float.md)
+DOUBLE | [Float64](../data_types/float.md)
+DATE | [Date](../data_types/date.md)
+DATETIME, TIMESTAMP | [DateTime](../data_types/datetime.md)
+BINARY | [FixedString](../data_types/fixedstring.md)
+
+All other MySQL data types are converted into [String](../data_types/string.md).
+
+[Nullable](../data_types/nullable.md) data type is supported.
```

Comment:
not yet, as you've found out?

Reply:
You can get the nullable fields from the MySQL table. This example is valid https://clickhouse.yandex/docs/en/operations/table_engines/mysql/#usage-example. You cannot insert the data in the table with nullable fields (or with the filelds that have `nullable` in their name).